### DISCUSSION TITLE
How to add SQL driver

### DISCUSSION THREAD

**Discussion by nschuering** [dataplane-app/dataplane#602] [Q&A]
How can ist add the MS-SQL ODBC Driver to the worker?

**Comment by saul-data** [comment]
Hi there @nschuering, how are you running the worker, is it inside the docker image? I’m assuming you are running this on Windows, that you’d like to use ODBC?

Typically, we use Python’s sqlalchemy to connect to databases. If that works then I can get you an example to use.
